crates/sui-indexer-alt/src/handlers/sum_coin_balances.rs

```diff
@@ -159,7 +152,7 @@ impl Handler for SumCoinBalances {
             }
         }
 
-        let update_chunks = updates.chunks(UPDATE_CHUNK_ROWS).map(|chunk| {
```

Comment:
added temp tests to ensure values are the same before and after, removed them to avoid overheard later on.